app/src/main/java/com/triviagenai/triviagen/trivia/presentation/TriviaQuestionViewModel.kt

```diff
@@ -100,12 +114,18 @@ class TriviaQuestionViewModel @Inject constructor(
                     currentState.copy(score = currentState.score + POINTS)
             }
             delay(2000)
-            nextQuestion()
+            nextQuestion(navController)
+            isOptionButtonEnabled = true
         }
     }
+
+
+    fun exitApp(activity: Activity) {
+        finishAffinity(activity)
```

Comment:
I believe that the Activity should not be passed to the ViewModel. Maybe you can manage the state from the ViewModel and execute this in a Composable?